packages/terra-form-select/src/combobox/Frame.jsx

```diff
@@ -536,8 +535,7 @@ class Frame extends React.Component {
    */
   role() {
     const { disabled } = this.props;
-    // role="application" needed to allow JAWS to work correctly with the select and use our key event listeners
```

Comment:
Looks like the role `application` was added for JAWS. Does it have any impact on JAWS response.

Reply:
Hi @supreethmr . We did not find any difference in JAWS after updating the role to combobox